### ISSUE TITLE
Feature: RPP Mobile Optimization changes for RPP PRODUCTION

### ISSUE THREAD

**Issue by dianamartin** [cityofaustin/atd-data-tech#5767]
1. Apply Permits page
- add rich text
- insert line break, separate Spanish line and English line
- Look into HTML buttons
- copy text, align right

2. Permit Information
- Label description "Look up address"
- insert line break before <aref> - looks better on the phone bc it splits without it

3.  Permit Information
- Vehicle Details
- Change layout from 2 columns to 1 column
- Create more "Hidden Field (For Formatting)" fields, add in between each license grouping

4. Required Documents
- Made Required Documents - bold (option 1)
- Create CSS to increase the size of Page Titles (option 2)
- Label description - add line break

5. Review Application 
- Switch email to column 1
- Switch Applicant Type to column 2
- Resident Address field - move label to top align
- Resident Address - max width of map 350

5.5 Edit App Form
- Email address field is not enlarging like the other fields - LOOK IN THIS
- Insert line break in front of "See RPP Eligibility Listing)
- Compare New Form v. Edit Form
- License Plate groupings (see notes from 7.)

6. Total Permit Costs - Review Page
- 1 column for every section
  - Resident Decal and Visitor Hangtag Cost
  - Resident Decal Cost
  - Visitor Hangtag Cost
  - Day Pass Cost
- second grouping with Total Cost by itself - or add new details and insert divider

7. Vehicle Information - Review page
- created 1 column
- license plate groupings with space between 

8. Digital Signature - Review page
- Add check box
- Add short text field for Name entry
- Remove electronic signature? 
  - what would we do for historical signatures? document retention

**Comment by ChrispinP** [comment]
@moorerst Discuss possible maintenance downtime with team and stakeholders for these mobile optimizations and couple it with the New Builder switchover

**Comment by moorerst** [comment]
@chrispin how much downtime will be needed for this maintenance?

**Comment by dianamartin** [comment]
@moorerst @ChrispinP cc: @johnclary I talked to John about this and we'll need to schedule time to do the form changes and also time to address unscheduled issues during maintenance to make sure we leave ourselves with buffer time. 

I've done something similar in the past, I can dig up the email I sent to internal stakeholders to see how long I allotted for the maintenance outage.

**Comment by dianamartin** [comment]
Email template to internal stakeholders

## Subject
### What's Happening - Friday, May 3rd, 3:00 pm to 5:30 pm -  Finance and Purchasing System MAINTENANCE OUTAGE

## Body
WHAT'S HAPPENING – TOMORROW, Friday, May 3rd after 3 pm
Need to apply application enhancement configurations. 
 
WHY ARE WE DOING THIS?
The new enhancement configurations will streamline the process for the Finance team so that they can process purchase requests efficiently. 
 
WHAT IS IMPACTED?
The following application: Finance and Purchasing - Production 

WHAT IS REQUIRED OF YOU?
Nothing. Please log off the Finance and Purchasing system by 3:00 PM on Friday, 5/3/2019. Your account will be temporarily disabled during this time and reactivated once maintenance work is completed.
 
HOW TO GET TECHNICAL SUPPORT?
If you have questions please email transportation.data email `[insert email address]` if you experience any issues after the maintenance work is completed.

**Comment by moorerst** [comment]
@ChrispinP @dianamartin based on our schedule can you propose several dates/times that I can present to the stakeholders?

**Comment by dianamartin** [comment]
cc: @moorerst

@ChrispinP - what's the estimate on the changes we have above? How long do you think it would take you do do all of those items uninterrupted in one day. I would then take that estimate and at 1.5 hours to it. From there we'd have to ask the RPP staff typically when they see the applications come in slower or we suggest an off-peak time like an early morning or late evening maintenance outage.

**Comment by moorerst** [comment]
emailed stakeholders to coordinate which day would be best for the maintenance window

**Comment by moorerst** [comment]
@dianamartin @ChrispinP per Steve Thursday perhaps. Fridays have ben busy lately for some reason.

**Comment by moorerst** [comment]
Planned for Thursday 4/22 - all day maintenance window for these changes and switchover to new Knack builder.
Will post maintenance downtime schedule on RPP Home page beginning 4/19

**Comment by johnclary** [comment]
Thanks, y'all. Is there a test environment where I can preview these changes?

**Comment by ChrispinP** [comment]
@johnclary yes, the March 26th RPP Test environment.

**Comment by johnclary** [comment]
This looks great! Small suggestion—when I was testing on mobile I was thrown off by the large "Permit Application Type" header on the second page of the app. I suggest we move this to a normal-sized label and put it above the address fields.

Before:
![Screen Shot 2021-04-14 at 12 39 27 PM](https://user-images.githubusercontent.com/14793120/114755751-9ee09f00-9d1f-11eb-9cd2-8ceb8bbc8da8.png)

After:
<img width="669" alt="Screen Shot 2021-04-14 at 12 47 09 PM" src="https://user-images.githubusercontent.com/14793120/114755745-9daf7200-9d1f-11eb-8593-e029385bea36.png">

**Comment by ChrispinP** [comment]
Mobile Styling is now in place. I was able to update it some to include new read-only name field and email fields.

New builder removes the ability to add the same field multiple times to a view so my blank formatting field is no longer usable and spacing is additionally adjusted between views since there is no longer groups. There could be some more spacing improvements but otherwise, the mobile application is in a good spot now.

Steph and I have completed successful tests. Ready for review for any further adjustments and double check our desktop application is still in a good spot.
